media/transcoder/create_job_from_template.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
I'd like to keep both since there are sections in the documentation devoted to each. Also, creating a job from a template requires more setup than from a preset.